src/components/Dashboard/ManageUsersPage/EditUser.js

```diff
@@ -31,7 +31,7 @@ class EditUser extends Component {
         firstName: '',
         lastName: '',
         role: 'floods_community_editor',
-        communityId: props.currentUser.communityId,
+        communityId: (props.currentUser.role === 'floods_super_admin' ? '9001' : props.currentUser.communityId),
```

Comment:
I see how this works but we should probably put a low priority issue in the backlog to make sure we don't always need to have a community with an ID of 9001 for super admins to work (not going to be an issue with our current setup but might cause issues in the future if we try using different datasets etc.)